src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1010,6 +1012,348 @@ static bool processAreaScope(Scope* scoP, BSONObj &areaQuery)
   return true;
 }
 
+/* ****************************************************************************
+*
+* qStringFilters -
+*
+* FIXME P9: this function currently abuses of the std::string() wrapping for char* in order to
+* make comparisons work. An smarter solution could be developed.
+*
+*/
+static void qStringFilters(std::string& in, std::vector<BSONObj> &filters)
```

Comment:
For something that big, it's better to wait.
We already have an issue about 'mongoBackend refactoring', right?

NTC (except for issue-creation if necessary)